components/healthcare/code/healthcare-bbe/why-ballerina-for-healthcare-apps.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: 'Why is Ballerina the way to write healthcare applications?'
-description: 'Java and HAPI have long been the de facto for health app development. However, cloud native healthcare application development goes beyond dealing with objects and frameworks. This domain needs a fresh architecture and technology that treats concepts like FHIR and HL7 as first class citizens in the language, alongside concepts like JSON and APIs.<br/><br/>
+title: 'What makes Ballerina the ideal choice for developing healthcare applications?'
+description: 'Java and [HAPI](https://hapifhir.io/) have long been the de facto for health app development. However, cloud-native healthcare application development goes beyond dealing with objects and frameworks. This domain needs a fresh architecture and technology that treats healthcare standards like FHIR(Fast Healthcare Interoperability Resources) and HL7 as first-class citizens in the language, alongside concepts like JSON and APIs.<br/><br/>
```

Comment:
Why we had to expand FHIR? If we really need to do that,  Then shouldn't it be `FHIR` within the brackets ? Also then why not HL7?

Reply:
Removed the full form.